asst1/asst1.py

```diff
@@ -0,0 +1,64 @@
+import collections
+import string
+
+def swapchars (w) :
+  temp_str = w.translate(None, string.punctuation)
+  temp_str = temp_str.translate(None, " ")
+  freq = collections.Counter(temp_str.lower()).most_common()
+  most, num = freq[0]
+  least, num1 = freq[len(freq) - 1]
+  print most
+  print least
+  ret = ""
+  for c in w :
+    if c == most.upper() :
+      ret += least.upper()
+    elif c == most.lower() :
+      ret += least.lower()
+    elif c == least.upper() :
+      ret += most.upper()
+    elif c == least.lower() :
+      ret += most.lower()
+    else :
+      ret += c
+  return ret
+
+def sortcat (n, *args) :
+  xs = []
+  str = ""
+  i = n
+  for a in args :
```

Comment:
this code is very C-style! Did you take CS50? Overall very good though -- we'l learn how to write code more "pythonically" throughout the comp (: